packages/WillowCreekApp/src/user-settings/Locations/index.js

```diff
@@ -22,10 +21,7 @@ const Location = (props) => (
         }}
         changeCampusOptions={{
           awaitRefetchQueries: true,
-          refetchQueries: [
-            { query: GET_FEED_FEATURES, variables: undefined },
-            { query: GET_CONTENT_CHANNELS, variables: undefined },
-          ],
+          refetchQueries: [{ query: GET_FEED_FEATURES, variables: undefined }],
```

Comment:
This might need to be updated to refer to the `GET_HOME_FEED` query

Reply:
good catch